services/cd-service/pkg/repository/transformer.go

```diff
@@ -431,6 +431,7 @@ func (c *CreateApplicationVersion) Transform(
 	transaction *sql.Tx,
 ) (string, error) {
 	version, err := c.calculateVersion(ctx, transaction, state)
+	fmt.Println(version)
```

Comment:
Forgotten debug print? If not, please wrap it in a descriptive message and use a proper logger.